managed/00_Group_birthdays_greeting_recipients.mgd.php

```diff
@@ -0,0 +1,23 @@
+<?php
+return [
+    [
+        'name' => 'Group_birthday_greeting_recipients',
+        'entity' => 'Group',
+        'cleanup' => 'always',
+        'update' => 'unmodified',
+        'params' => [
+            'version' => 4,
+            'values' => [
+                'name' => 'birthday_greeting_recipients_group',
+                'title' => ts('Birthday greeting recipients group'),
+                'description' => ts('Every contact in this group is elected to get a birthday email if birthday is set correctly'),
+                'data_type' => NULL,
+                'is_reserved' => TRUE,
+                'is_active' => TRUE,
+                'is_locked' => TRUE,
+                'option_value_fields' => ['name', 'label', 'description', 'icon'],
+            ],
+        ],
+        'match' => ['name'],
+    ]
+];
```

Comment:
How to trigger this when updating an extension?

Reply:
Needs to be added to the upgrade routine. See [here](https://github.com/systopia/de.systopia.remoteevent/blob/92e1661fb79ce8ef1d3003c7bdacf13033c54753/CRM/Remoteevent/Upgrader.php#L161) for an example.

Reply:
The example given by @pbatroff does not use managed entities, but an implementation of the [systopia/Custom-Data-Helper](https://github.com/systopia/Custom-Data-Helper) class.

Managed entities should be synchronised when installing the extension automatically (and also on cache clears), but might need a Civix mixin for the definition to be picked up by that routine.